### DISCUSSION TITLE
WASD shouldn't trigger other shortcuts while in freelook

### DISCUSSION THREAD

**Discussion by wgetJane** [godotengine/godot-proposals#8973] [Editor]
When you have the animation editor open and try to move a 3D camera with freelook by holding down RMB and using WASD, it will probably cause an animation to be played

This is because the A and D keys are bound to play the current animation in the animation editor

I think that this is unwanted behaviour because keyboard shortcuts shouldn't trigger two different things in different contexts simultaneously

**Comment by Sauermann** [comment]
This is more of a bug than a proposal.

**Comment by wgetJane** [comment]
now that you mention it, i should've made an issue instead

**Reply by Calinou** [reply]
Issue opened: https://github.com/godotengine/godot/issues/87639
